### ISSUE TITLE
Hyperparameter Tuning

### ISSUE THREAD

**Issue by imchandanmohan** [imchandanmohan/smart-resume-builder#16]
Try different learning rates, dropout, batch size.
